scripts/link-cli.cjs

```diff
@@ -4,8 +4,31 @@ const fs = require('fs');
 const path = require('path');
 
 try {
-  const script = process.argv[2] || '/usr/local/bin/agoric';
+  const script = process.argv[2] || `${process.env.HOME || '/usr/local'}/bin/agoric`;
```

Comment:
Is there a presumption here that a user's personal local executables are in a directory named `bin`?  Because mine aren't.

Reply:
Not really a presumption, just a default.  You can override the default with an argument (which power users probably will do).